sigh/lib/sigh/runner.rb

```diff
@@ -327,7 +327,7 @@ def certificates_to_use
         UI.important("Found more than one code signing identity. Choosing the first one. Check out `fastlane sigh --help` to see all available options.")
         UI.important("Available Code Signing Identities for current filters:")
         certificates.each do |c|
-          str = ["\t- Name:", c.display_name, "- ID:", c.id + " - Expires", c.expires.strftime("%d/%m/%Y")].join(" ")
+          str = ["\t- Name:", c.display_name, "- ID:", c.id + " - Expires", c.expiration_date.strftime("%d/%m/%Y")].join(" ")
```

Comment:
I tested out this change and it didn't quite work. It seems that `c.expiration_date` is a `String`, not a `DateTime`, so you still get an error. I updated my local `runner.rb` file with the following line instead to make it work:
```
str = ["\t- Name:", c.display_name, "- ID:", c.id + " - Expires", DateTime.iso8601(c.expiration_date).strftime("%d/%m/%Y")].join(" ")
```

Reply:
Thanks for spotting this! 
I wrote a test to prevent this from happening: 

<img width="596" alt="image" src="https://user-images.githubusercontent.com/8419048/102702782-4ef83c80-4245-11eb-8ce1-0caef1d9d80d.png">

💪

Reply:
I added some commits - could you try again if you have a chance? 😃